src/buttons.cpp

```diff
@@ -102,15 +102,20 @@ commandRaw Buttons::getCommandRaw() {
   case commandRaw::pauseLong: longPressFactor = 0xff;
                               ignoreRelease = true;
                               break;
+#ifdef FIVEBUTTONS
   case commandRaw::upLong   :
   case commandRaw::downLong : if (!settings.invertVolumeButtons) longPressFactor = 0xff; else ++longPressFactor;
```

Comment:
In the branch 'issue_028' is a bug since my last commit. I removed the dependency from settings.invertVolumeButtons for FIVEBUTTONS but did not changed it here. So here should be used 
`case commandRaw::downLong : longPressFactor = 0xff;`